e2eAIOK/DeNas/scores/compute_de_score.py

```diff
@@ -59,7 +59,7 @@ def do_compute_nas_score(model_type, model, resolution, batch_size, mixup_gamma,
     if model_type == "cnn":
         nas_score, score, latency = do_compute_nas_score_cnn(model_type, model, resolution, batch_size, mixup_gamma, expressivity_weight, complexity_weight, diversity_weight, saliency_weight, latency_weight)
         return nas_score, score, latency
-    elif model_type == "transformer" or model_type == "bert" or model_type == "asr":
+    elif model_type == "transformer" or model_type == "bert" or model_type == "asr" or model_type == "hf":
```

Comment:
@tianyil1 @Peach-He , if only CNN is using a different descore, can we use this score as default so we don't need to add so much model_types here?

Reply:
I have refined it. Thanks for your comments.